packages/taquito-local-forging/src/constants.ts

```diff
@@ -5,6 +5,8 @@
 
 import { pad } from './utils';
 
+export const ENTRYPOINT_MAX_LENGTH = 31;
```

Comment:
Add a comment with a link to https://tezos.gitlab.io/protocols/005_babylon.html#transactions-now-have-an-entrypoint

useful to the reader to easily jump to the definition.